examples/counter/temp/api.purs

```diff
@@ -0,0 +1,70 @@
+module App where
```

Comment:
The temp file is where the intermediary Purescript file is rendered to. We could remove that altogether, but it's probably more useful to see what servant-purescript compiles to on the way.

For the sake of clarity, I might rename that directory to `src` or something and drop other Purescript application code in there.